--- conf/carbon.conf.example.orig	2011-04-21 14:56:25.000000000 -0400
+++ conf/carbon.conf.example	2011-04-21 14:56:54.000000000 -0400
@@ -1,10 +1,10 @@
 [cache]
-LOCAL_DATA_DIR = /opt/graphite/storage/whisper/
+LOCAL_DATA_DIR = /var/spool/carbon/whisper/
 
 # Specify the user to drop privileges to
 # If this is blank carbon runs as the user that invokes it
 # This user must have write access to the local data directory
-USER = 
+USER = carbon 
 
 # Limit the size of the cache to avoid swapping or becoming CPU bound.
 # Sorts and serving cache queries gets more expensive as the cache grows.
