--- carbon-0.9.12.imeyer/conf/carbon.conf.example	2013-08-21 16:52:35.000000000 +0000
+++ carbon-0.9.12/conf/carbon.conf.example	2013-09-15 18:15:37.323000878 +0000
@@ -30,13 +30,20 @@
 #
 #LOCAL_DATA_DIR = /opt/graphite/storage/whisper/
 
+STORAGE_DIR    = /var/lib/carbon/
+LOCAL_DATA_DIR = /var/lib/carbon/whisper/
+WHITELISTS_DIR = /var/lib/carbon/lists/
+CONF_DIR       = /etc/carbon/
+LOG_DIR        = /var/log/carbon/
+PID_DIR        = /var/run/
+
 # Enable daily log rotation. If disabled, a kill -HUP can be used after a manual rotate
 ENABLE_LOGROTATION = True
 
 # Specify the user to drop privileges to
 # If this is blank carbon runs as the user that invokes it
 # This user must have write access to the local data directory
-USER =
+USER = carbon
 #
 # NOTE: The above settings must be set under [relay] and [aggregator]
 #       to take effect for those daemons as well
